doc/source/dev/missing-bits.rst

```diff
@@ -0,0 +1,260 @@
+        stats.zmap(scores, compare, nan_policy='raise')
+
+The ``match`` argument ensures that the test doesn't pass by raising
+a ``ValueError`` that is not related to the input containing ``nan``.
+
+
+A parameter called lambda
+~~~~~~~~~~~~~~~~~~~~~~~~~
+There are functions where convention dictates that (ideally) a parameter
+should be called 'lambda'.  In Python, 'lambda' is a reserved word that
+cannot be used as a variable name.  If a new function or class has a
+parameter where a name that is similar to 'lambda' is really the best
+choice, the preferred name to use is [TBD: ``lam``, ``lmb``, ``lamb``,
+``lmbda``, ``lambda_``?].
+
+There has been a lack of consistency in the past:
+
+============================== =============
+Function or class              Variable name
+============================== =============
+`scipy.stats.boxcox`           ``lmbda``
+`scipy.stats.boxcox_llf`       ``lmb``
+`scipy.stats.yeojohnson`       ``lmbda``
+`scipy.stats.yeojohnson_llf`   ``lmb``
+`scipy.stats.chi2_contingency` ``lambda_``
+`scipy.stats.boltzmann`        ``lambda_``
+`scipy.stats.planck`           ``lambda_``
+`scipy.stats.tukeylambda`      ``lam``
+`scipy.signal.cspline1d`       ``lamb``
+`scipy.signal.qspline1d`       ``lamb``
+`scipy.signal.cspline2d`       ``lambda`` [*]
+`scipy.signal.qspline2d`       ``lambda`` [*]
+`scipy.signal.spline_filter`   ``lmbda``
+`scipy.special.tklmbda`        ``lmbda``
+`scipy.special.boxcox`         ``lmbda``
+`scipy.special.boxcox1p`       ``lmbda``
+`scipy.special.inv_boxcox`     ``lmbda``
+`scipy.special.inv_boxcox1p`   ``lmbda``
+============================== =============
+
+.. [*] This function accepts positional arguments only, but the docstring
+       documents the argument as `lambda`.
+
+
+Implicit string continuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When an implicit continuation is used to create a long literal string, the
+string is generally split at a space, and the space is included in the
+second part of the string.
+
+  Yes::
+
+      long_string = ("This is a very long string that demonstrates"
+                     " this guideline.")
+
+  No::
+
+      long_string = ("This is a very long string that demonstrates "
+                     "this guideline.")
+
+
+Documentation Guidelines
+------------------------
+
+Use of LaTeX
+~~~~~~~~~~~~
+[TBD.  In the past, we had an unwritten rule that LaTex markup should be
+restricted to the Notes section.  We don't follow that rule these days.
+Should we?]
+
+
+Use "must", not "should"
+~~~~~~~~~~~~~~~~~~~~~~~~
+When specifying a required condition on the input parameters, the
+word "must" is preferable to "should".  For many English speakers,
+"must" implies a stronger constraint than "should",  e.g. "I must
+have oxygen to live" versus "I should exercise more".
+
+    Yes::
+
+            Parameters
+            ----------
+            x : float
+                x must be nonnegative.
```

Comment:
Might be work talking about how to format the params in the description vs others things with double backticks.
```suggestion
                `x` must be nonnegative.
```